tests/test_config_gsobject.py

```diff
@@ -595,6 +595,7 @@ def test_realgalaxy():
                  },
         'gal5' : { 'type' : 'RealGalaxy' , 'index' : 23, 'noise_pad_size' : 10 },
         'gal6' : { 'type' : 'RealGalaxyOriginal' },
+        'gal7' : { 'type' : 'RealGalaxy' , 'random' : True}
```

Comment:
I guess I don't have any tests here of config builds of COSMOSGalaxy.  Probably worth adding a few, including the new random index feature.  (Looks like I use COSMOSGalaxy in a few tests in `test_config_image.py`, but they don't really test for accuracy.)